cubicchunks-api/src/main/java/io/github/opencubicchunks/cubicchunks/api/worldgen/CubePrimer.java

```diff
@@ -21,29 +21,46 @@
  *  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  *  THE SOFTWARE.
  */
-package cubicchunks.worldgen.generator;
+package io.github.opencubicchunks.cubicchunks.api.worldgen;
 
 import mcp.MethodsReturnNonnullByDefault;
 import net.minecraft.block.Block;
 import net.minecraft.block.state.IBlockState;
+import net.minecraft.init.Blocks;
 
 import javax.annotation.Nonnull;
 import javax.annotation.ParametersAreNonnullByDefault;
 
 @ParametersAreNonnullByDefault
 @MethodsReturnNonnullByDefault
-public class CubePrimer implements ICubePrimer {
+public class CubePrimer {
+    public static final IBlockState DEFAULT_STATE = Blocks.AIR.getDefaultState();
 
     private final char[] data = new char[4096];
 
-    @Override
+    /**
+     * Get the block state at the given location
+     *
+     * @param x cube local x
+     * @param y cube local y
+     * @param z cube local z
+     *
+     * @return the block state
+     */
     public IBlockState getBlockState(int x, int y, int z) {
         @SuppressWarnings("deprecation")
         IBlockState iblockstate = Block.BLOCK_STATE_IDS.getByValue(this.data[getBlockIndex(x, y, z)]);
```

Comment:
`GameData.getBlockStateIDMap().getByValue(this.data[getBlockIndex(x, y, z)])`
Do a same thing, but does not cause deprecation warning.

Reply:
Well this is the way vanilla did it, and I found no other examples at that time. And `GameData.getBlockStateIDMap()` probably gives the same object anyway. If you want to fix such minor things, submit a PR after this is finished.

Reply:
No, I don't really care. Just want you to know, because I recently use this.